pkg/ipcache/metadata.go

```diff
@@ -274,16 +305,18 @@ func FilterMetadataByLabels(filter labels.Labels) []string {
 	return matching
 }
 
-// RemoveLabelsFromIPs wraps RemoveLabels to provide a convenient method for
-// the caller to remove all given prefixes at once. This function will trigger
-// policy update and recalculation if necessary on behalf of the caller if any
-// changes to the kube-apiserver were detected.
+// removeLabelsFromIPs provides a convenient method for the caller to remove
+// all given prefixes at once. This function should be the inverse of
+// InjectLabels(). This function will trigger policy update and recalculation
+// if necessary on behalf of the caller.
```

Comment:
super minor nit, the extra words provide no additional meaning:

```suggestion
// removeLabelsFromIPs removes
// all given prefixes at once. This function should be the inverse of
// InjectLabels(). This function will trigger policy update and recalculation
// if necessary on behalf of the caller.
```

Reply:
Thanks, this should have been changed when I unexported this funciton.